src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -50,8 +50,7 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         Upcoming upcoming = ParserUtil.parseUpcoming(argMultimap.getValue(PREFIX_UPCOMING).orElse(""));
-        LastContact lastcontact = ParserUtil.parseLastContact(argMultimap.getValue(PREFIX_LASTCONTACT)
-                .orElse(""));
+        LastContact lastcontact = ParserUtil.parseLastContact(argMultimap.getValue(PREFIX_LASTCONTACT).orElse(""));
```

Comment:
Nice QOL change

Reply:
i think @SimHanKiong 's point here cannot be understated.

Reply:
Thanks bro